spec/requests/api/v2/users_spec.rb

```diff
@@ -85,9 +85,9 @@ def perform_request
       end
       let(:user_params)           { params.dig :_jsonapi, :data, :attributes }
       let(:accept_terms_date_utc) { DateTime.now.utc }
-      let(:email)                 { 'dave.smith@gmail.com' }
-      let(:first_name)            { 'Dave' }
-      let(:last_name)             { 'Smith' }
+      let(:first_name)            { Faker::Name.first_name }
+      let(:last_name)             { Faker::Name.unique.last_name }
+      let(:email)                 { Faker::Internet.unique.email("#{first_name}.#{last_name}") }
```

Comment:
You're missing a `unique` on the `first_name`. :) I still think you should just have this:

```
let(:user) { create :user, first_name: 'Bob', last_name: 'Kim', email: 'bob.kim@local' }
```

inside this block, to circumvent Faker for this test.

Reply:
Opps this simple `unique` thing is getting tenacity 

I settling for 

- user factory to using unique, due to the fact that building a list of user there maybe a clash
- for the test setup for setting a particular user details using unique ie first_name, last_name, email

vs hardcoding the user variable and/or the individual variables of first_name, last_name, email as that introduce confidence by lack of variation vs confidence by various

I care about mechanic of the action and that the original and intended is not the same.
I don't care explicitly what has changed

Reply:
@defong The point is that this is a much smaller change that clearly communicates what the test is trying to do. Changing the names and emails to use Faker only communicates that they're meant to be different from "Dave Smith" as a side-effect.

Reply:
Apologies, point taken, was seething with Mr Lottery and Agent Smith and my jackpot.